sevntu-checkstyle-sonar-plugin/src/main/resources/com/github/sevntu/checkstyle/sonar/checkstyle-extensions.xml

```diff
@@ -377,6 +377,20 @@
     </param>
   </rule>
 
+  <rule>
+    <key>com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodCheck</key>
+    <name>Forbid Certain Method</name>
+    <category name="coding"/>
+    <description>Forbids certain method in class.</description>
+    <configKey>Checker/TreeWalker/com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodCheck</configKey>
+    <param key="methodName" type="REGULAR_EXPRESSION">
+      <description>Regex to match method name.</description>
+    </param>
+    <param key="argumentCount" type="STRING">
+      <description>Number or range to match number of arguments. Multiple numbers/ranges must be comma separated.</description>
+    </param>
```

Comment:
if params have defaults, it s should be placed here. See example: https://github.com/raghavgautam/sevntu.checkstyle/blob/190cc01523f7411f217252f0c100960fe7cf74bf/sevntu-checkstyle-sonar-plugin/src/main/resources/com/github/sevntu/checkstyle/sonar/checkstyle-extensions.xml#L11

Reply:
Done.